pybossa/api/__init__.py

```diff
@@ -186,8 +186,7 @@ def new_task(project_id):
 def _retrieve_new_task(project_id):
 
     project = project_repo.get(project_id)
-
-    if project is None:
+    if project is None or not project.published:
```

Comment:
should this exclude admins and owners?

Reply:
i think no, as admin/owner would be able to submit via "test it" project option

Reply:
doesn't "test it" go through the `newtask` route too?

Reply:
ah right. will fix. thanks